resources/js/Pages/Layout.vue

```diff
@@ -115,6 +116,10 @@ export default defineComponent({
             },
         },
     },
+    mounted() {
+        (this.resizeObserver as unknown as ResizeObserver) = new ResizeObserver(this.onWindowResize);
+        (this.resizeObserver as unknown as ResizeObserver).observe(this.$refs.contentWrap as Element);
```

Comment:
If you accept the suggestion above, then this will become

```suggestion
        this.resizeObserver  = new ResizeObserver(this.onWindowResize);
        this.resizeObserver.observe(this.$refs.contentWrap as Element);
```